### PR TITLE
Remove deprecated "instantiate" syntax with capitalized Type or Value

### PR DESCRIPTION
**Pull request by SkySkimmer**

### CODE REVIEW

**Review by herbelin — APPROVED** [review]
OK, see also lint.

**Review by ppedrot — APPROVED** [review]

File: doc/tools/docgram/fullGrammar
Comment (Zimmi48) [inline doc/tools/docgram/fullGrammar:1918]: Can you look into why this syntax is reintroduced by the SSR plugin?

Comment (SkySkimmer) [reply doc/tools/docgram/fullGrammar:1918]: from https://github.com/coq/coq/blob/7e4a34f67ad993fc3742cbb1600b330b3d31a7c6/plugins/ssr/ssrvernac.mlg#L350-L358
no idea what this code is for
cc @coq/ssreflect-maintainers

Comment (gares) [reply doc/tools/docgram/fullGrammar:1918]: that may be bacuse of is a kwd and hence ssr did repair the orig grammar.
please remove.

**Comment by Zimmi48** [comment]
You have a warning for an unused `open Ltac_plugin.Extraargs` that makes several jobs fail.

**Review by Zimmi48 — APPROVED** [review]

**Comment by ppedrot** [comment]
Will merge when CI finishes, it's quite a simple PR.

**Comment by ppedrot** [comment]
@coqbot merge now

**Comment by CohenCyril** [comment]
Wasn't bigenough in Coq CI?

**Comment by Zimmi48** [comment]
Apparently not

**Pull request merged**
